src/main/java/com/blibli/future/vo/ReportVo.java

```diff
@@ -0,0 +1,243 @@
+package com.blibli.future.vo;
+
+import java.io.Serializable;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class ReportVo implements Serializable{
+	private static final long serialVersionUID = 2006346112358122493L;
```

Comment:
Which one is better: `ReportVO`or `ReportResponse` or `ReportResponseVO`?
What's your opinion @sebastianbagya @aamesaa ?